play/api/generators.py

```diff
@@ -10,16 +10,16 @@
 
 
 def new_text(  # pylint: disable=too-many-arguments
-    words="",
-    x=0,
-    y=0,
-    font=None,
-    font_size=50,
-    color="black",
-    angle=0,
-    transparency=100,
-    size=100,
-):
+    words: str = "",
+    x: int = 0,
+    y: int = 0,
+    font: str = "C:/location/to/font_file.ttf",
```

Comment:
We use None for a reason here to load, the default font. Since not everyone uses windows I would recommend using a more general file path like /path/to/file